edx_credentials_themes/templates/mitpe/credentials/programs/d670134d-3155-4b8a-b058-4618bb2b3e2c/certificate.html

```diff
@@ -0,0 +1,9 @@
+{% extends "../base.html" %}
+{% load staticfiles %}
+
+{% block top_logo %} 
+<img class="organization-logo" src="{% static 'mitpe/images/mitxpro-logo.png' %} " alt="MIT XPro logo">
+{% endblock %}
+
+{% block course_dates %} March 20, 2017 - August 7, 2017 {% endblock %}
```

Comment:
Make place holders solely for both start and end date. The rest of the styling (e.g. ` - `) belongs in the base template. Doing this will make it very easy to replace these placeholders with values from the server later.